### PR TITLE
Unit Test Issue2006b

### PR DESCRIPTION
**Pull request by sir-braz**
npm run jest

### CODE REVIEW

File: website/src/components/Buttons/LikertButtons.tsx [hunk 1]
```diff
@@ -25,7 +25,17 @@ export const LikertButtons = ({
...
+            key={value}
+            id={`radio-${value}`}
+            value={value}
+            borderColor="gray.400"
+            data-cy="radio-option"
+            size="md"
+            padding="0.6em"
+          >
+            <label htmlFor={`radio-${value}`}>{value}</label>
+          </Radio>
```
Comment (notmd) [inline website/src/components/Buttons/LikertButtons.tsx:38]: This change may not be correct. We may have multiple elements with the same id when rendering many `LikertButtons` component

File: website/src/components/Buttons/LikertButtons.tsx
Comment (notmd) [inline website/src/components/Buttons/LikertButtons.tsx:1]: This component is used in many places. Why did you delete it?

**Comment by notmd** [comment]
Most of the tested components are just static components. I'm not sure if these tests will have many benefits or just create noise.

File: website/src/components/Buttons/LikertButtons.tsx
Comment (sir-braz) [reply website/src/components/Buttons/LikertButtons.tsx:1]: I'm going to reverse, Because what I did was wrong, I thought I had to delete what I did

**Comment by sir-braz** [comment]
@notmd But are the tests working?
